### PR TITLE
[CodeQuality] Add SimplifyArrayCallableRector

### PR DESCRIPTION
**Pull request by TomasVotruba**
Ref #424

### CODE REVIEW

File: packages/CodeQuality/src/Rector/FuncCall/SimplifyArrayCallableRector.php [hunk 1]
```diff
@@ -25,6 +25,12 @@
     private $functionsWithCallableArgumentPosition = [
         'array_filter' => 1,
         'array_map' => 0,
+        'array_walk' => 1,
+        'array_reduce' => 1,
+        'usort' => 1,
+        'uksort' => 1,
+        'uasort' => 1,
+        'array_walk_recursive' => 1,
```
Comment (carusogabriel) [inline packages/CodeQuality/src/Rector/FuncCall/SimplifyArrayCallableRector.php]: We should list all of them. I'll provide a full list later

Comment (TomasVotruba) [reply packages/CodeQuality/src/Rector/FuncCall/SimplifyArrayCallableRector.php]: I tried to Google them, but dunno how to find them all. 

Thanks :+1:

**Comment by TomasVotruba** [comment]
I update DOCS just once before release, so I can focus on development itself.

**Comment by carusogabriel** [comment]
This will need to do the opposite, see https://github.com/FriendsOfPHP/PHP-CS-Fixer/pull/4050

**Comment by TomasVotruba** [comment]
Both are fine as personal preferences might differ.

Could you send the invereted one?

**Comment by carusogabriel** [comment]
@TomasVotruba Sure. Next week remind me to discuss with you about configs for our rectors

**Pull request merged**